### PR TITLE
Add default user mode

### PR DESCRIPTION
**Pull request by in0ni**
Basic user-mode mapping

### CODE REVIEW

**Comment by jdugan6240** [comment]
Would you mind adding a blurb to the README about this and what the bindings are? If the user mode is there, it stands to reason that users should know about it. 🙂

File: rc/kak-dap.kak
Comment (jdugan6240) [inline rc/kak-dap.kak]: I'd get rid of this. The point of `dap-evaluate` is to send an expression to the debug adapter, so it can evaluate said expression and show the output to the user. This can't be done with a fixed keybinding, as the expression evaluated (in this case, none) will always be the same.

**Review by jdugan6240 — CHANGES_REQUESTED** [review]

File: rc/kak-dap.kak
Comment (raiguard) [reply rc/kak-dap.kak]: The binding doesn't actually execute the command - notice the lack of `<ret>`. It simply populates your command bar with the command name, then lets you type the rest.

In that vein, I don't think it should have the space between the colon and command, because you probably want it to stay in your command history.

Comment (jdugan6240) [reply rc/kak-dap.kak]: Ah - I didn't notice that. Never mind then.

**Comment by in0ni** [comment]
All set:

* added blurb on README, and also noted requirements (update wording as you wish)
* keep dap-evaluate in history

**Comment by jdugan6240** [comment]
I may want to change the wording of the user mode docs in the README, but otherwise this should do fine.

One question, though, before I merge: this project's development mainly happens on Codeberg, and this Github project is primarily a mirror for visibility purposes. Since Codeberg doesn't (yet) have a Contributors view, would you mind me putting you in a "Contributors" section in the README?

**Comment by in0ni** [comment]
Yes, please update the text as you see fit. And yes -- you may add me to "Contributors", there are a few other fixes/changes I am looking to implement. I can run them by you later, to see if you are OK with these. I'll create a ticket.

I don't have a Codeburg account, and hopefully I can continue with github for now, if you don't mind.

**Comment by jdugan6240** [comment]
You can continue contributing with Github - that's not a problem. I have both Github and Codeberg as remotes locally, so syncing them up is as simple as pulling from one remote and pushing to the other.

I'm at work at the moment, so I can't merge this right now, but I will when I get home.

**Comment by in0ni** [comment]
Absolutely no rush. Honestly I've only invested time on this due to my joy for hacking and interest in kakoune. I'm pretty much saturated with work/screen time myself.

**Pull request merged**
